app/uk/gov/hmrc/gform/sharedmodel/formtemplate/DateExpr.scala

```diff
@@ -89,13 +89,15 @@ object OffsetYMD {
 }
 
 sealed trait DateExprValue {
-  def toLocalDate =
+  def toLocalDate(maybeFormStartDate: Option[LocalDate]): LocalDate =
```

Comment:
As I said previously, this should accept `formStartDate: LocalDate`. There is no reason to use `Option` for value which will always exists.